### PR TITLE
first version of very simple lesson 3

### LINKED ISSUE #6: Episode 3: What are the SCRUM basics?
# Objectives:
* Describe the 3 key roles in a Scrum team
* List the different scrum rituals
* List the Scrum artifacts
* Formulate a user story

Use resource 2!

### PR DESCRIPTION
**Pull request by cwmeijer**
which is a link to a course mainly

Sven update: I implemented my own comments. @bpmweel can you review?

### CODE REVIEW

File: _episodes/03-SCRUM-basics.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: "What is SCRUM?"
```
Comment (svenvanderburg) [inline _episodes/03-SCRUM-basics.md]:
```suggestion
title: "Introduction to SCRUM?"
```
Maybe?

File: _episodes/03-SCRUM-basics.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+teaching: 10
+exercises: 10
```
Comment (svenvanderburg) [inline _episodes/03-SCRUM-basics.md]: This needs an update

File: _episodes/03-SCRUM-basics.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+questions:
+- "What are the 3 key roles in a Scrum team?"
+- "What are the different scrum rituals?"
+- "What are the Scrum artifacts?"
+objectives:
+- "List benefits of working in a team"
+keypoints:
+- "If done well, working in a team is fun and productive"
```
Comment (svenvanderburg) [inline _episodes/03-SCRUM-basics.md]: This needs an update!

File: _episodes/03-SCRUM-basics.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+---
+Go to http://growingagile.thinkific.com/courses/scrumbasics and enroll.
```
Comment (svenvanderburg) [inline _episodes/03-SCRUM-basics.md]: We point to enrolment in the setup of the course! See #7

File: _episodes/03-SCRUM-basics.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+Follow this ~1 hour course. It includes 3-8 minute video's focusing on 
+various SCRUM topics, as well as accompanying exercises.
+
```
Comment (svenvanderburg) [inline _episodes/03-SCRUM-basics.md]: Maybe include a discussion with some reflection on what we learned. How does it help me when working in a team?

**Review by svenvanderburg — COMMENTED** [review]
Nice start! Indeed this is not much more than a pointer to the course. We can add a little bit more I think :)

File: _episodes/03-SCRUM-basics.md (hunk 1 above)
Comment (svenvanderburg) [reply _episodes/03-SCRUM-basics.md]: Done.

File: _episodes/03-SCRUM-basics.md (hunk 2 above)
Comment (svenvanderburg) [reply _episodes/03-SCRUM-basics.md]: Updated.

File: _episodes/03-SCRUM-basics.md (hunk 3 above)
Comment (svenvanderburg) [reply _episodes/03-SCRUM-basics.md]: Updated.

File: _episodes/03-SCRUM-basics.md (hunk 4 above)
Comment (svenvanderburg) [reply _episodes/03-SCRUM-basics.md]: Updated.

File: _episodes/03-SCRUM-basics.md (hunk 5 above)
Comment (svenvanderburg) [reply _episodes/03-SCRUM-basics.md]: Added a discussion.

**Review by JaroCamphuijsen — APPROVED** [review]
This looks like a good first step towards an episode, however in my opinion this will not finish episode 3. What I would think would fit well in this episode is: 

1. logical linking to other episodes at least the previous one. So a short explanation about how the scrum framework relates to agile (the previous episode).
2. Some content and/or exercises about how you can translate various concepts from scrum (customers, product, etc.) to an academic situation (professors, students, papers, projects, etc.).

This can be addressed in a different PR. Created a new issue #27 to work on episode 3.

**Pull request merged**